.github/actions/build/action.yml

```diff
@@ -0,0 +1,16 @@
+name: 'Install, build & cache'
```

Comment:
The different between this and `setup`, is that this is meant to create the cache, while the setup task now requires the cache.